common/common.go

```diff
@@ -537,6 +537,25 @@ func (e *multiError) Is(incoming error) bool {
 	return false
 }
 
+// ExcludeError returns a new error without any errors.Is(err)
```

Comment:
> without any errors.Is(err)

I don't understand since you use it twice in this function

Reply:
Is this clearer:
```
// ExcludeError returns a new error excluding any errors matching excl
// For a standard error it will either return the error unchanged or nil
// For multiError it will remove any errors matching excl and return the remaining errors or nil
```

Fixed 5770e95e

Please reopen if that didn't help. I might need some more help understanding what's not clear.